packages/application-shell-connectors/src/components/application-context/index.ts

```diff
@@ -1,8 +1,14 @@
-import { ProviderProps } from './application-context';
+import {
+  ProviderProps,
+  TApplicationContext as ApplicationContext,
+} from './application-context';
 
 export type TProviderProps<
   AdditionalEnvironmentProperties extends {}
 > = ProviderProps<AdditionalEnvironmentProperties>;
+export type TApplicationContext<
+  AdditionalEnvironmentProperties extends {}
+> = ApplicationContext<AdditionalEnvironmentProperties>;
```

Comment:
As a reminder, because we're using babel with typescript, we need to re-export the type as an alias.